components/faction/wikis/stormgate/faction_data.lua

```diff
@@ -36,13 +38,24 @@ local factionProps = {
 }
 
 return {
-	factionProps = factionProps,
+	defaultGame = Info.defaultGame,
+	factionProps = {
+		[Info.defaultGame] = factionProps,
+	},
 	defaultFaction = 'u',
-	factions = {'v', 'i', 'r', 'u'},
-	knownFactions = {'v', 'i', 'r'},
-	coreFactions = {'v', 'i'},
-	aliases = {
-		human = 'v',
-		host = 'i',
+	factions = {
+		[Info.defaultGame] = {'v', 'i', 'r', 'u'},
 	},
+	knownFactions = {
+		[Info.defaultGame] = {'v', 'i', 'r'},
+	},
+	coreFactions = {
+		[Info.defaultGame] = {'v', 'i'},
+	},
```

Comment:
would break sc2, sc, warcraft and stormgate

they get used directly
maybe we could keep them as is on non aoe and have them different on aoe???

would need to adjust ~50 modules, guessing around half off them non git

Reply:
out of curiosity: where would any of those even be used on aoe?
the usages i can think of would access the functions in `Module:Faction` and not those arrays (or in case of aoe tables of arrays)

Reply:
They aren't used on AoE, that's why i proposed the option to leave them non-game-specific in the PR description

Reply:
The question is more whether the other exports, factions and aliases are used elsewhere i guess - as these would be required to be adjusted.

Reply:
`Faction.factions` is used (hence included in the code snippet i commented on^^)

aliases aren't

Reply:
for `Faction.factions`:
- https://liquipedia.net/stormgate/Module:MatchGroup/Util/Custom (open PR)
- https://liquipedia.net/warcraft/Module:MatchGroup/Util/Custom
- https://liquipedia.net/warcraft/Module:ParticipantTable/Custom
- https://liquipedia.net/commons/Module:MatchGroup/Util/Starcraft/downstream (non-git)
- https://liquipedia.net/commons/Module:MatchGroup/Util/Starcraft
- https://liquipedia.net/commons/Module:ParticipantTable/Starcraft

core, known, etc are used more (especially the non git usage is way higher too)
aliases doesn't have usage as of now to my knowledge

Reply:
I'll add the changes for `factions`->`getFactions()` to this PR.
The downstream module reports no global usages, but i'll change it once this is merged.

core/known will stay unchanged

Reply:
just checked usage locally on sc/sc2: none
moved to archive, so ignore the downstream one